olc_art.cpp

```diff
@@ -869,41 +867,28 @@ olc_db::try_get_result_type olc_db::try_get(detail::art_key k) const noexcept {
     return {};
     // LCOV_EXCL_STOP
   }
-
-  auto node{root.load()};
-
-  if (UNODB_DETAIL_UNLIKELY(node == nullptr)) {
-    if (UNODB_DETAIL_UNLIKELY(!parent_critical_section.try_read_unlock())) {
-      // LCOV_EXCL_START
-      spin_wait_loop_body();
-      return {};
-      // LCOV_EXCL_STOP
-    }
+  detail::olc_node_ptr node{root.load()};
+  if (UNODB_DETAIL_UNLIKELY(node == nullptr)) {  // special path if empty tree.
+    if (UNODB_DETAIL_UNLIKELY(!parent_critical_section.try_read_unlock()))
+      return {};  // LCOV_EXCL_LINE
+    // return an empty result (breaks out of caller's while(true) loop)
```

Comment:
This change is suspicious in the following sense.  For a TSan build, the spin_wait_loop_body() does 
```
std::this_thread::yield();
```
By removing that line here the scheduling for TSan builds is affected.

Reply:
Will remove this from master